samples/v2/two_step_pipeline_containerized.py

```diff
@@ -28,23 +27,34 @@ def component1(text: str, output_gcs: Output[Dataset]):
     return ContainerSpec(
         image='google/cloud-sdk:slim',
         command=[
-            'sh -c | set -e -x', 'echo', text, '| gsutil cp -', output_gcs.uri
+            'sh',
+            '-c',
+            'echo "$0" | gsutil cp - "$1"',
```

Comment:
We have lots of examples that show using `gsutil`, but I actually don't think this is best practice, as the component should only have to interact with the container's local filesystem.

For example, the following `ContainerSpec` here would achieve the same effect, without using `gsutil`:

```python
    dsl.ContainerSpec(
        image='alpine',
        command=['sh', '-c', 'mkdir --parents $(dirname "$1") && echo "$0" > "$1"',],
        args=[text, output_gcs.path])
```

This allows the KFP runtime to copy the artifact from the container filesystem to remote storage. It is also file store provider agnostic.

WDYT?

Reply:
Thanks for the suggestion! The new pipeline works well and is updated in the test files.